File: backend/Gs-Data/com/src/main/java/Gs_Data/project/com/Entities/Notification.java
package Gs_Data.project.com.Entities;


import jakarta.persistence.*;
import lombok.Getter;
import lombok.Setter;

import java.time.LocalDateTime;


@Getter
@Setter
@Entity
public class Notification {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;
    private Long userId;          // ID de l'utilisateur lié
    private String title;         // Titre de la notification
    private String message;       // Message principal
    private String status;        // "READ" ou "UNREAD"
    private LocalDateTime date;   // Date de création
}

File: frontend/app/src/main/java/iset/dsi/myapplication/Notification.kt
package iset.dsi.myapplication
/*
data class Notification(
    val id: Long,
    val userId: Long,
    val message: String,
    val status: String,  // "READ" ou "UNREAD"
    val date: String
)
*/


data class Notification(
    val message: String,
    val status:String,
    val title: String,
    val description: String,
    val date: String
)
